### ISSUE TITLE
After Update from 4.0.9 to 4.1.2 I got a strange error message

### ISSUE THREAD

**Issue by schluckspecht** [softsyst/qirx_issues#118]
After update from 4.0.9 to 4.1.2 I got a strange error message. I chose the option that the old config should not be deleted to prevent licence issue . Windows = Win 11.

The path exists. "C:\Program Files\softsyst\QIRX4\qirx_audio\qirx_audio.exe"

![image](https://github.com/softsyst/qirx_issues/assets/84860874/7585bd03-4242-4e56-b648-591e7be4c55a)

If I double click on this qirx_audio it starts:
QIRX AAC stream player V2.2
Copyright (c) Clem Schmidt, 2019-2021, qirx.softsyst.com

libfaad2 is Copyright (C) 2003-2005 M. Bakker, Nero AG, http://www.nero.com

NAudio is an open source .NET audio library written by Mark Heath (mark.heath@gmail.com)
For more information, visit http://github.com/naudio/NAudio
PortAudio Portable Real-Time Audio Library, Copyright (c) 1999-2011 Ross Bencina and Phil Burk
PortAudio P/Invoke interfacing based on work by Atsushi Eno, https://github.com/atsushieno/portaudio-sharp

 The following UDP Addresses and ports are used:
        AAC Streams:    Remote endpoint:         127.0.0.1:8766
        AAC Streams:    Own endpoint:            127.0.0.1:8767
        Commands:       Remote endpoint:         127.0.0.1:8768
        Commands:       Own endpoint:            127.0.0.1:8769

        Import Resolver for assembly qirx_audio, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null created.

UDP connected to: 127.0.0.1:8766
UDPCmd connected to: 127.0.0.1:8768
Stream Receive thread entered.
Command thread entered.
Waiting for command...

**Comment by superblyb** [comment]
The new version is working fine with me, but I deleted the old config.

**Comment by schluckspecht** [comment]
Yes, the only solution was a complete deinstallation. I saved the  qirx4.config, deinstalleted the whole programn, installed it again and started it. Then I got this licence error of course and I closed the program. Then I copied the backuped qirx4.config to "C:\Users\\%user%\AppData\Local\qirx4" and started quirx again, and it works without any error. That means, the update option with "keep the config" does not work.
